sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -66,8 +66,8 @@ class CertificateClient(AsyncKeyVaultClientBase):
         :param bool enabled: Determines whether the object is enabled.
         :param tags: Application specific metadata in the form of key-value pairs.
         :type tags: dict(str, str)
-        :returns: The created CertificateOperation
-        :rtype: coroutine
+        :returns: A coroutine for the creation of the certificate. Awaiting the coroutine
+        returns the created Certificate if creation is successful, the CertificateOperation if not.
```

Comment:
I'd link to `Certificate` and `CertificateOperation` here.

Reply:
ok, switchin!